src/components/ChatForm/ChatForm.test.tsx

```diff
@@ -127,9 +127,9 @@ describe("ChatForm", () => {
     const textarea = app.container.querySelector("textarea")!;
     await user.type(textarea, "foo");
     await user.keyboard("{Enter}");
-    const epexted = `@file ${activeFile.path}:${activeFile.line1}-${activeFile.line2}\n@symbols-at ${activeFile.path}:${activeFile.cursor}\nfoo\n`;
+    const expected = `@file ${activeFile.path}:${activeFile.line1}-${activeFile.line2}\n@symbols-at ${activeFile.path}:${activeFile.cursor}\nfoo\n`;
```

Comment:
should line1 and line2 have been replaced with the cursor?

Reply:
oh wait this test is skipped anyway since the `Lookup symbols` was removed.

Reply:
Eeyup